backend/src/test/java/com/pickpick/message/application/ReminderServiceTest.java

```diff
@@ -61,293 +67,361 @@ class ReminderServiceTest {
+            }
+
+            return messagesInChannel;
+        }
+
+        private List<Reminder> saveRemindersAtDifferentTimes(final Member member, final List<Message> messages) {
+            List<Reminder> savedReminders = new ArrayList<>();
+            int totalSize = messages.size();
+
+            for (int i = 0; i < totalSize; i++) {
+                LocalDateTime remindDateTime = LocalDateTime.now().plusHours(i + 1);
+                Reminder reminder = reminders.save(
+                        new Reminder(member, messages.get(totalSize - i - 1), remindDateTime));
+                savedReminders.add(reminder);
+            }
+
+            return savedReminders;
+        }
+
+        private Message saveDummyMessage(final Member member, final Channel channel) {
+            Message message = new Message(UUID.randomUUID().toString(), "리마인더 설정 용 메시지", member, channel,
+                    LocalDateTime.now(), LocalDateTime.now());
+            return messages.save(message);
+        }
+
+        private List<Long> extractIds(final List<Reminder> reminders) {
+            return reminders.stream()
+                    .map(Reminder::getId)
+                    .collect(Collectors.toList());
+        }
+
+        private List<Long> extractOrderedByRemindDateIds(final List<Reminder> reminders) {
+            return reminders.stream()
+                    .sorted(Comparator.comparing(Reminder::getRemindDate))
+                    .map(Reminder::getId)
+                    .collect(Collectors.toList());
+        }
+
+        private List<Long> extractOrderedByRemindDateIdsDefaultLimit(final List<Reminder> reminders) {
+            return reminders.stream()
+                    .sorted(Comparator.comparing(Reminder::getRemindDate))
+                    .map(Reminder::getId)
+                    .limit(20)
+                    .collect(Collectors.toList());
+        }
+
+        private List<Long> extractFutureFromTargetOrderedByRemindDateIds(final List<Reminder> reminders,
+                                                                         final Reminder target, final int limit) {
+            return reminders.stream()
+                    .filter(reminder -> reminder.getRemindDate().isAfter(target.getRemindDate()))
+                    .sorted(Comparator.comparing(Reminder::getRemindDate))
+                    .map(Reminder::getId)
+                    .limit(limit)
+                    .collect(Collectors.toList());
+        }
     }
 
-    @DisplayName("다른 사용자의 리마인더 삭제시 예외")
-    @Test
-    void deleteOtherMembers() {
-        // given & when & then
-        assertThatThrownBy(() -> reminderService.delete(1L, 1L))
-                .isInstanceOf(ReminderDeleteFailureException.class);
+    @DisplayName("리마인더 저장/수정/삭제 시")
+    @Nested
+    class other {
+
+        @AfterEach
+        void tearDown() {
+            databaseCleaner.clear();
+        }
+
+        @DisplayName("리마인더 저장 시 새로운 리마인더를 저장한다")
+        @Test
```

Comment:
```suggestion

        @DisplayName("리마인더 저장 시 새로운 리마인더가 DB에 추가된다(?)")
        @Test
```

얘도 별로인 것 같은데 이 DisplayName 봤을 때 `새로운 리마인더`가 뭐지?? 라는 생각이 들었어서 개선할 수 있지 않을까? 해서 리뷰 남격요!

Reply:
그냥 `리마인더를 저장한다`나 `리마인더 저장 요청 시 리마인더가 저장된다` 같은 것도 괜찮을 것 같아요!

Reply:
`리마인더를 저장한다`로 변경했습니다